stitch/cli.py

```diff
@@ -46,18 +48,39 @@ def enhance_args(to, no_standalone, no_self_contained, extra_args):
 )
 @click.pass_context
 @click.argument('input_file', type=click.File('rb'))
+@click.option('-y', '--yaml', type=click.File('rb'), default=None)
 @click.option('-o', '--output_file', type=str, default=None)
 @click.option('-t', '--to', default=None)
 @click.option('--no-standalone', is_flag=True, default=False,
               help='Produce a document fragment instead.')
 @click.option('--no-self-contained', is_flag=True, default=False,
               help='Use external files for resources like images.')
-def cli(ctx, input_file, output_file, to, no_standalone, no_self_contained):
+def cli(ctx, input_file, yaml, output_file, to, no_standalone, no_self_contained):
     if to is None:
         to = infer_format(output_file)
     input_text = input_file.read().decode('utf-8')
+    input_file_path = os.path.abspath(input_file.name)
+    yaml_text = None
+    if yaml is not None:
+        yaml_text = yaml.read().decode('utf-8')
     extra_args = enhance_args(to, no_standalone, no_self_contained, ctx.args)
-    convert(input_text, to, output_file=output_file, extra_args=extra_args)
+
+    # Atom compatibility patch: for some reason RTerm.exe does not close
```

Comment:
At the moment there is no issue opened. I've started writing Atom package that is a simple GUI wrapper around Stitch. And I eventually came across this problem. I haven't came with any nice ideas of tracking processes from Node. The only Node solution was: create a pause in Stitch when every child processes are spawned (like RTerm.exe) - then I should tell Node to track processes - then pause ends and Stitch closes - then Node kills processes if there is still any. But it's a bad design - better kill processes right in python.

I guess it would be better first to try close them via Jupyter means in Stitch (I do not know whether it is done or not at the moment). And if it's not succeded then kill them via psutil. But this would require some investigations. So for now I've decided to go with rough and ready psutil only solution.

This actually isn't Node bug. Maybe it's a RTerm bug. Maybe it's Jupyter bug, maybe it's Stitch lack of processes management (haven't looked at this part of the Stitch at all). But the main suspect is RTerm definitely.

Fixing it at RTerm side is not universal solution because Stitch can use any Jupyter kernels. There are lots of them. And this bug can appear again on other kernel. So there should be process killer inside Stitch in order to use it in Atom package. Actually when using Stitch in command prompt the RTerm process is active only until command prompt is closed. So it's not a problem at all.

The bug is on Windows 7 x64, installed [R](https://cran.r-project.org/) and [IRkernel](https://irkernel.github.io/installation/).